robottelo/libvirt_discovery.py

```diff
@@ -56,7 +56,13 @@ def __init__(
         else:
             self.image_dir = image_dir
         if mac is None:
-            self.mac = gen_mac(multicast=False, locally=True)
+            # fe:* MAC range is considered reserved in libvirt
+            while True:
+                mac = gen_mac(multicast=False, locally=True)
+                if not mac.startswith(u'fe'):
```

Comment:
@rochacbruno but the purpose of the fauxfactory is to generate a mac addresses. Those with FE: are perfectly valid. I decided to go this way so we keep the fauxfactory lightweight and not overweight its methods by further options. This way we might go with other methods as well.